cli/azd/internal/global_command_options.go

```diff
@@ -20,4 +20,7 @@ type GlobalCommandOptions struct {
 	// AZURE_DEV_COLLECT_TELEMETRY is set to 'no'.
 	// Defaults to true.
 	EnableTelemetry bool
+
+	// When supported by a command has the specified output format. ex) JSON, table, None.
+	Output string
```

Comment:
I'm a little hesitant about this.. I think ideally the approach would be to add an `outputFlag` similar to `envFlag` and enable cmd flags to attach an `outputFlag` if an output format is supported.

Reply:
Not used anymore - removed.